src/metrics.go

```diff
@@ -232,6 +232,32 @@ func getDbAndKeys(keysFlag sdkArgs.JSON) map[string][]string {
 	return databaseKeys
 }
 
+// getRenamedCommands returns a map of command and renamed command pair
+// Example: '{"CONFIG":"ZmtlbmZ3ZWZl-CONFIG"}'
+func getRenamedCommands(renamedCommandsFlag sdkArgs.JSON) map[string]string {
```

Comment:
I don't feel very comfortable having more and more args-related functions in the `metrics.go` file.
It might be better to create a new `args.go` file and put everything arg-related there. It would still be the same package so no big changes would be required.

Reply:
moved args related functions to `args.go`